app/src/main/java/com/framgia/moviedbtraining/rest/RequestHelper.kt

```diff
@@ -13,11 +13,12 @@ import retrofit2.Response
  */
 object RequestHelper {
 
-  fun getNowPlaying(page: Int, mViewModel: NowPlayingContractNew.ViewModel): List<Movie>? {
+  fun getNowPlaying(type: String, page: Int,
```

Comment:
why change api request to get movie by type but keep this method name is nowplaying???